sp800-63b/cover.md

```diff
@@ -3,7 +3,7 @@
 
 # <a name="800-63b"></a> DRAFT NIST Special Publication 800-63B
 
-# Digital Authentication Guideline
+# Digital Identity Guidelines
 
 ### Authentication and Lifecycle Management
```

Comment:
Since we're in the business of renaming, any reason 'Authenticator' wouldn't make the cut?  'Authenticators and Lifecycle management'?  I don't feel strongly.